packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -15,8 +15,128 @@
 
 library amplify_authenticator;
 
-/// A Calculator.
-class Calculator {
-  /// Returns [value] plus 1.
-  int addOne(int value) => value + 1;
+import 'package:flutter/material.dart';
+
+//State
+import 'package:amplify_authenticator/src/state/inherited_forms.dart';
+import 'package:amplify_authenticator/src/state/inherited_auth_bloc.dart';
+import 'package:amplify_authenticator/src/state/inherited_auth_viewmodel.dart';
+
+//Screens
+import 'package:amplify_authenticator/src/screens/loading_screen.dart';
+import 'package:amplify_authenticator/src/screens/signin_screen.dart';
+import 'package:amplify_authenticator/src/screens/signup_screen.dart';
+import 'package:amplify_authenticator/src/screens/confirm_signup_screen.dart';
+
+//Bloc
+import 'package:amplify_authenticator/src/blocs/auth/auth_data.dart';
+import 'package:amplify_authenticator/src/blocs/auth/auth_bloc.dart';
+
+//Views
+import 'package:amplify_authenticator/src/views/signin_viewmodel.dart';
+import 'package:amplify_authenticator/src/views/signup_viewmodel.dart';
+import 'package:amplify_authenticator/src/views/confirm_signup_viewmodel.dart';
+
+//Services
+import 'package:amplify_authenticator/src/services/amplify_auth_service.dart';
+
+//Widgets
+import 'package:amplify_authenticator/src/widgets/forms.dart';
+import 'package:amplify_authenticator/src/widgets/default_forms.dart';
+
+//Exports
+export 'package:amplify_authenticator/src/widgets/forms.dart';
+export 'package:amplify_authenticator/src/widgets/form_fields.dart';
+export 'package:amplify_authenticator/src/widgets/buttons.dart'
+    show SignOutButton;
+
+/// Authenticator
+// ignore: must_be_immutable
+class Authenticator extends StatefulWidget {
+  /// Constructor
+  Authenticator(
+      {required this.child, signInForm, signUpForm, confirmSignUpForm}) {
+    this.signInForm = signInForm ?? DefaultForms.signInForm();
+    this.signUpForm = signUpForm ?? DefaultForms.signUpForm();
+    this.confirmSignUpForm =
+        confirmSignUpForm ?? DefaultForms.confirmSignUpForm();
+  }
+
+  ///Custom Form
```

Comment:
You can add some more explanatory documentation here, or possibly in the constructor.

See [here](https://dart.dev/guides/language/effective-dart/documentation) for more guidance.